chart/hooks/pre-upgrade/remove-deployment-updater-readiness.sh

```diff
@@ -18,8 +18,18 @@ spec:
       - name: cc-deployment-updater-cc-deployment-updater
         readinessProbe: ~
 '
+set +o pipefail
+scheduler_list=$(kubectl get statefulsets --namespace "$NAMESPACE" | grep scheduler | cut -d " " -f 1)
+set -o pipefail
 
-kubectl patch statefulset --namespace "$NAMESPACE" scheduler --patch "$patch"
+if [ "${scheduler_list}" == "" ]; then
+  echo "No scheduler statefulset found."
+  exit 0
+fi
+
+for i in ${scheduler_list}; do
```

Comment:
I don't care too much, but if you already change the PR, why not use a more descriptive name than `i`? I think `scheduler` would be an obvious choice.

Reply:
@jandubois, thanks. I update the code based on your comments.